src/cli/commands/test/iac-local-execution/policy-engine.ts

```diff
@@ -3,18 +3,34 @@ import {
   IacFileData,
   IacFileScanResult,
   PolicyMetadata,
+  EngineType,
 } from './types';
 import { loadPolicy } from '@open-policy-agent/opa-wasm';
 import * as fs from 'fs';
-import * as path from 'path';
+import { getLocalCachePath, LOCAL_POLICY_ENGINE_DIR } from './local-cache';
 
-const LOCAL_POLICY_ENGINE_DIR = `.iac-data`;
-const LOCAL_POLICY_ENGINE_WASM_PATH = `${LOCAL_POLICY_ENGINE_DIR}${path.sep}policy.wasm`;
-const LOCAL_POLICY_ENGINE_DATA_PATH = `${LOCAL_POLICY_ENGINE_DIR}${path.sep}data.json`;
+export async function getPolicyEngine(
+  engineType: EngineType,
+): Promise<PolicyEngine> {
+  return (
+    policyEngineCache[engineType] ||
+    (policyEngineCache[engineType] = await buildPolicyEngine(engineType))
```

Comment:
Suggestion: To make this optional assignment more explicit
```suggestion
    if (policyEngineCache[engineType]) {
      return policyEngineCache[engineType]
    }
    policyEngineCache[engineType] = await buildPolicyEngine(engineType)
    return policyEngineCache[engineType]
```

Reply:
I like shorthand syntax and minimum amount of return statements  🤓 
but ok, changing